.github/workflows/docker-publish-releasing.yml

```diff
@@ -6,62 +6,56 @@ on:
     branches:
       - master
       - release-*
-    
+
     paths:
       - releasing/VERSION
-  
+
   # Run tests for any PRs.
   pull_request:
 
 env:
   IMAGE_NAME: models-web-app
 
 jobs:
-  # Run tests.
-  test:
-    runs-on: ubuntu-latest
-    
-    steps:
-      - uses: actions/checkout@v2
-      
-      - name: Run tests
-        run: |
-          docker build . --file Dockerfile
-
   # Push image to GitHub Container Registry.
   push:
-    # Ensure test job passes before pushing image.
-    needs: test
-    
     runs-on: ubuntu-latest
+    # Only run on push events, not PRs to the trigger branches/paths
     if: github.event_name == 'push'
     permissions:
       packages: write
       contents: read
-    
+
     steps:
-      - uses: actions/checkout@v2
-      
-      - name: Build image
-        run: |
-          docker build . --file Dockerfile --tag ${IMAGE_NAME}
-      
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
       - name: Log into GitHub Container Registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      
-      - name: Push image
-        run: |
-          IMAGE_ID=ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME
-          
-          # Change all uppercase to lowercase
-          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
-          VERSION=$(cat ./releasing/VERSION)
-          
-          echo IMAGE_ID=$IMAGE_ID
-          echo VERSION=$VERSION
-          docker tag $IMAGE_NAME $IMAGE_ID:$VERSION
-          docker push $IMAGE_ID:$VERSION
\ No newline at end of file
+
+      - name: Extract metadata (tags, labels) for Docker
```

Comment:
I think the formal term is OCI instead of Docker. Docker is just one of many OCI implementations. Podman, crio, containerD etc. So please use the OCI naming in the descriptions